Kudu.Contracts/Functions/FunctionSecretsJsonOps.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Core.Functions
+{
+    public class FunctionSecretsJsonOps : IKeyJsonOps<FunctionSecrets>
+    {
+        public int GetKeyNumbers()
+        {
+            return 1;
+        }
+
+        // have the schema related info enclosed in this class
+        public string GenerateKeyJson(Tuple<string,string>[] keyPair, out string unencryptedKey)
+        {
+            unencryptedKey = keyPair[0].Item1;
+            return JObject.Parse($"{{\"keys\":[{{\"name\":\"default\",\"value\":\"{keyPair[0].Item2}\",\"encrypted\": true }}]}}").ToString(Formatting.Indented);
```

Comment:
Should be able to simplify this so we don't go from string to JObject just to get a formatted string.